modules/nf-core/pretextsnapshot/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: "pretextsnapshot"
+description: a module to generate images from Pretext contact maps.
+keywords:
+  - pretext
+  - image
+  - hic
+  - PNG
+  - JPG
+  - BMP
```

Comment:
I _think_ we prefer lower case... but I'm not 100% sure so you can ignore if you wish ;)

Reply:
I can make the lower-case, no prob.

Reply:
in the pretextsnapshot case, the prefix is the prefix used to name the files (in cases with more than one output file: like ${prefix}_${sequenceID}.png ).  For the $outdir i tend to use the current working directory, but I vaguely remember someone wanting the option to specify it, so I made it a task.ext .

Reply:
personally i alsop don't use the --prefix option at all, as I am happy if it names them by sequence_id